source/postprocess/velocity_boundary_statistics.cc

```diff
@@ -22,6 +22,7 @@
 
 #include <aspect/postprocess/velocity_boundary_statistics.h>
 #include <aspect/utilities.h>
+#include <aspect/geometry_model/interface.h>
```

Comment:
why do you need these includes?

Reply:
aspect/geometry_model/interface.h used to be included with aspect/utilities.h but no longer is after this change.